plugin/trino-mongodb/src/main/java/io/trino/plugin/mongodb/MongoTableHandle.java

```diff
@@ -32,8 +32,8 @@
 {
     private final SchemaTableName schemaTableName;
     private final RemoteTableName remoteTableName;
-    private final TupleDomain<ColumnHandle> constraint;
     private final Optional<String> filter;
+    private final TupleDomain<ColumnHandle> constraint;
```

Comment:
Can we reword the commit message as `Make variable arrangement consistent in MongoTableHandle`